libkbfs/interfaces.go

```diff
@@ -1679,6 +1695,9 @@ type MutableBareRootMetadata interface {
 	// key generation.
 	GetUserDeviceKeyInfoMaps(keyGen KeyGen, extra ExtraMetadata) (
 		readers, writers UserDeviceKeyInfoMap, err error)
+	// OnRekeyDone is called after all rekeying work has been performed on the underlying
```

Comment:
I think a better name would be `FinalizeRekey` or something, since `OnRekeyDone` sounds like an event notification.

Also, can you pass in a `Crypto` instead of a `Config`? Eventually I'd like to move `RootMetadata` and such to its own package.

Also, please name the parameters here, for consistency.